Packs/GCP/Integrations/GCP/GCP.yml

```
@@ -1479,6 +1479,610 @@ script:
+      type: string
+    - contextPath: GCP.Compute.Zones.deprecated.obsolete
+      description: An optional RFC3339 timestamp on or after which the state of this resource is intended to change to OBSOLETE. This is only informational and the status will not change unless the client explicitly changes it.
+      type: string
+    - contextPath: GCP.Compute.Zones.deprecated.deleted
+      description: An optional RFC3339 timestamp on or after which the state of this resource is intended to change to DELETED. This is only informational and the status will not change unless the client explicitly changes it.
+      type: string
+    - contextPath: GCP.Compute.Zones.region
+      description: Full URL reference to the region which hosts the zone.
+      type: string
+    - contextPath: GCP.Compute.Zones.selfLink
+      description: Server-defined URL for the resource.
+      type: string
+    - contextPath: GCP.Compute.Zones.availableCpuPlatforms
+      description: Available cpu/platform selections for the zone. Do not use field = 7 or field = 11. Next available field = 14.
+      type: string
+    - contextPath: GCP.Compute.Zones.kind
+      description: Type of the resource. Always compute#zone for zones.
+      type: string
+
+# gcp-compute-networks-list
+  - arguments:
+    - description: GCP project ID.
+      name: project_id
+      required: true
+    - defaultValue: '500'
+      description: 'The maximum number of results per page that should be returned. If the number of available results is larger than maxResults, Compute Engine returns a nextPageToken that can be used to get the next page of results in subsequent list requests. Acceptable values are 0 to 500, inclusive. (Default: 500).'
+      name: maxResults
+    - description: A filter expression that filters resources listed in the response. The expression must specify the field name, a comparison operator, and the value that you want to use for filtering. The value must be a string, a number, or a boolean. The comparison operator must be either =, !=, >, or <.  For example, if you are filtering Compute Engine instances, you can exclude instances named example-instance by specifying name != example-instance.
+      name: filters
+    - description: Sorts list results by a certain order. By default, results are returned in alphanumerical order based on the resource name.  You can also sort results in descending order based on the creation timestamp using orderBy="creationTimestamp desc". This sorts results based on the creationTimestamp field in reverse chronological order (newest result first). Use this to sort resources like operations so that the newest operation is returned first.
+      name: orderBy
+    - description: Specifies a page token to use. Set pageToken to the nextPageToken returned by a previous list request to get the next page of results.
+      name: pageToken
```

```suggestion
      name: page_token
```
Please update the argument name in all of the relevant places.